### ISSUE TITLE
Full Android TV Support

### ISSUE THREAD

**Issue by LinkofHyrule89** [jpd002/Play-#81]
At the current time there isn't an Android TV specific interface and because Android TV doesn't have a menu button there is no way to change any of the settings. It would be preferable to have a box art based file picker similar to how PPSSPP has theirs setup. Full controller support up to however many controllers the original PS2 supported is also needed.

**Comment by adriandennis17** [comment]
is this a feature request ?
the emulator is still in early stages and some device's are not supported
features will slowly be available and more device's will be supported
as the development continues..at the moment their's only one dev..so you just have be patient.

**Comment by LinkofHyrule89** [comment]
I know I already talked to the developer about this I posted this purely to be on the issues list so people know it's on the roadmap. But as far as things go it's something that needs to be worked on early in development because otherwise it will be more of a hassle to go back and redesign. - Matthew Garbett

**Comment by adriandennis17** [comment]
oh..didn't know it was you matt..k.

**Comment by MisterHubs** [comment]
Speaking of a roadmap, wouldn't it be effective if there actually was one jotted down in the Wiki?

**Comment by jpd002** [comment]
@MisterHubs Good idea, I'm going to add a roadmap page, should help people see where the emulator is heading.

**Comment by LinkofHyrule89** [comment]
It's always a good idea to have a roadmap of where things are headed. Maybe other developers will want to help with the project and work on different parts of it.

**Comment by LinkofHyrule89** [comment]
I submitted two patches to help get this started keep in mind I don't have things setup to compile so you'll have to test the patches but one is just an image and the other is simply to get the app to show up on the Android TV Leanback Launcher. I made a temp banner by using the icon as a base it's probably pretty terrible but gets the job done for now.

**Comment by LinkofHyrule89** [comment]
Basic Android TV support has been added however there is still no way to access the options on Android TV at least not on the Nexus Player. I've made the developer aware of this issue but just wanted to relay the information here.
